ios/Podfile.lock

```diff
@@ -2330,8 +2346,10 @@ SPEC REPOS:
     - CocoaLumberjack
     - HMSegmentedControl
     - JitsiWebRTC
+    - libwebp
     - SDWebImage
     - SDWebImageSVGCoder
+    - SDWebImageWebPCoder
```

Comment:
This was added in this expo-image [PR].(https://github.com/expo/expo/pull/35802)